tests/src/Functional/ConfigurationPermissionTest.php

```diff
@@ -104,42 +84,35 @@ protected function assertPaths(bool $access) {
       $this->assertEquals($expected_code, $this->getSession()->getStatusCode(), $path);
     };
 
-    // General settings related admin pages.
-    $visit_path('/admin/config/apigee-edge');
-    $visit_path('/admin/config/apigee-edge/error-page-settings');
-    $visit_path('/admin/config/apigee-edge/settings');
-    $visit_path('/admin/config/apigee-edge/connection-config');
-
-    // Developer entity related admin pages.
-    $visit_path('/admin/config/apigee-edge/developer-settings');
-    $visit_path('/admin/config/apigee-edge/developer-settings/attributes');
-    $visit_path('/admin/config/apigee-edge/developer-settings/caching');
-    $visit_path('/admin/config/apigee-edge/developer-settings/sync');
-
-    if ($access) {
-      list($schedule_path, $schedule_query) = $this->findLink('Background');
-      list($run_path, $run_query) = $this->findLink('Now');
-      $visit_path($schedule_path, $schedule_query);
-      $visit_path($run_path, $run_query);
-    }
-    else {
-      $visit_path('/admin/config/apigee-edge/sync/schedule');
-      $visit_path('/admin/config/apigee-edge/sync/run');
+    // Get all routes defined by the module and check every route that requires
+    // the permission "administer apigee edge".
+    $module_path = $this->container->get('module_handler')->getModule('apigee_edge')->getPath();
```

Comment:
Wooot!!! :) Cool, I never thought that my idea could be solved this easily in Drupal 8.